config.txt

```diff
@@ -1,9 +1,15 @@
-core_freq=400
-dtoverlay=spi0-1cs
+gpu_mem=32
 dtparam=i2c_arm=on
+dtparam=i2s=on
 dtparam=spi=on
 dtparam=audio=on
-avoid_warnings=1
+dtoverlay=i2c-gpio
+dtoverlay=dwc2,dr_mode=host
```

Comment:
USB host mode is for 4g radios etc?

Reply:
It might not be needed, actually it probably doesn't even really work on some units due to a USB hardware design issue.

Reply:
Guess there's no harm in keeping it anyway